source/fundamentals/auth.txt

```diff
@@ -443,39 +422,35 @@ use the ``MongoCredential.createAwsCredential()`` factory method
 and add the ``MongoCredential`` instance to your ``MongoClient`` as shown
 in the following example:
 
-.. literalinclude:: /includes/fundamentals/code-snippets/MongoDbAwsAuth.java
-   :language: java
-   :dedent:
-   :start-after: start mongoCredential
-   :end-before: end mongoCredential
+.. literalinclude:: /examples/generated/AuthTest.snippet.aws-credential-mongo-cred.kt
+   :language: kotlin
    :emphasize-lines: 1
 
 If you need to specify an AWS session token, pass it to the
 `withMechanismProperty() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/MongoCredential.html#withMechanismProperty(java.lang.String,T)>`__
 method as shown in the following example:
 
-.. literalinclude:: /includes/fundamentals/code-snippets/MongoDbAwsAuth.java
-   :language: java
-   :dedent:
-   :start-after: start mongoCredentialSessionTokenCredential
-   :end-before: end mongoCredentialSessionTokenCredential
-   :emphasize-lines: 1,7
+.. literalinclude:: /examples/generated/AuthTest.snippet.aws-credential-session-token.kt
+   :language: kotlin
+   :emphasize-lines: 1, 2, 10
 
 To refresh your credentials, you can declare a ``Supplier`` lambda expression
 that returns new credentials as shown in the following example:
 
-.. literalinclude:: /includes/fundamentals/code-snippets/MongoDbAwsAuth.java
-   :language: java
-   :dedent:
-   :start-after: start refreshCredentials
-   :end-before: end refreshCredentials
-   :emphasize-lines: 4-5,9
+.. literalinclude:: /examples/generated/AuthTest.snippet.aws-refresh-credentials.kt
+   :language: kotlin
+   :emphasize-lines: 4-6, 9
 
 .. note::
```

Comment:
Suggestion: move this out of the note into reg text

Reply:
Done